use-cases/rag-on-gke/embedding-models/text-embedding/src/requirements.txt

```diff
@@ -0,0 +1,6 @@
+Flask
+requests
+sentence_transformers>=2.7.0
+waitress
```

Comment:
Can you pin your versions so that it is reproducible.

Reply:
and google's practise is to pin to the hashes: https://pip.pypa.io/en/stable/topics/secure-installs/

Reply:
fixed now